Update metadata-service.yaml

fpetke opened the pull request:
Use 1.*.* tags for the main Metadata Service, and 0.*.* tags for the test version.

ktatarnikovhiro commented on integration/apps/minio-tenant.yaml:
```diff
@@ -1,536 +1,202 @@
 metadata:
-  name: minio-tenant
-  namespace: argocd
+  name: glaciation-env-configuration
```
I am looking and see that this minio-tenant is very different from the one deployed from helm chart. Is there a way to deploy it from helm chart? if there is no way then I propose to make a separate tenant for data wrapping.

matthewrossi replied:
I managed to make it work with the Helm chart

ktatarnikovhiro commented on integration/apps/minio-tenant.yaml:
```diff
@@ -1,536 +1,202 @@
+  annotations:
+    # NOTE: To make sure the certificate is trusted by clients we should use
+    # something like Let's Encrypt
+    cert-manager.io/cluster-issuer: "private-ca-issuer"
```
I am afraid TLS is not available in the integration cluster in ingresses and ssl-redirect as well.

matthewrossi replied:
We could do TLS termination at ingress (like above), and then export the service without TLS, but I would suggest to go for this solution only after checking the current ingress does not work with the integration cluster.

ktatarnikovhiro replied:
I am merging it and we will see...

fpetke committed:
Update metadata-service.yaml

Use 1.*.* tags for the main Metadata Service, and 0.*.* tags for the test version.
